### PR TITLE
create markup

### PR DESCRIPTION
**Pull request by Maxus94**

### CODE REVIEW

File: src/components/UserTabs/UserCard/UserCard.jsx [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
-      </ButtonLink>
-      <ButtonLink
-        type="button"
-        color="light"
-        size="small"
-        onClick={console.log}
-      >
-        Following
-      </ButtonLink>
+      <div>
```
Comment (Iryna-Holova) [inline src/components/UserTabs/UserCard/UserCard.jsx:6]: too many divs, only li outside as container for content

File: src/components/UserTabs/ListItems/ListItems.jsx [hunk 2]
```diff
@@ -1,5 +1,13 @@
-const ListItems = () => {
-  return <div>ListItems</div>;
+import UserCard from '../UserCard/UserCard';
+
+const ListItems = ({ users, tabType }) => {  
+  return (
+    <ul>
+      {users.map(({id, name, avatar_preview, recipes}) => (
+        <li key={id}><UserCard name={name} avatar_preview={avatar_preview} recipes={recipes} tabType = {tabType} /></li>
```
Comment (Iryna-Holova) [inline src/components/UserTabs/ListItems/ListItems.jsx:7]: li should be inside UserCard
pass the whole user to UserCard

File: src/components/UserTabs/UserCard/UserCard.jsx [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
+              color="light"
+              size="small"
+              onClick={console.log}
+            >
+              Following
+            </ButtonLink>
+          )}
+        </div>
+      </div>
+      <ul>{recipes.map(recipy => <li key={recipy.id}><img src={recipy.preview} alt="" /></li>)}</ul>
```
Comment (Iryna-Holova) [inline src/components/UserTabs/UserCard/UserCard.jsx:33]: recipe)
empty alt

Comment (Maxus94) [reply src/components/UserTabs/UserCard/UserCard.jsx:33]: corrected

File: src/components/UserTabs/ListItems/ListItems.jsx (hunk 2 above)
Comment (Maxus94) [reply src/components/UserTabs/ListItems/ListItems.jsx:7]: corrected

**Review by Iryna-Holova — APPROVED** [review]

**Pull request merged**